numpy/ma/core.py

```diff
@@ -5898,10 +5843,39 @@ def __reduce__(self):
         """
         return (self.__class__, ())
 
+    def __iadd__(self, other):
+        return self
+    def __isub__(self, other):
+        return self
+    def __imul__(self, other):
+        return self
+    def __idiv__(self, other):
+        return self
+    def __ifloordiv__(self, other):
+        return self
+    def __itruediv__(self, other):
+        return self
+    def __ipow__(self, other):
+        return self
 
 masked = masked_singleton = MaskedConstant()
+    
+def returnMasked(out):
```

Comment:
Probably better to use `return_masked`. Camel case is a rarity in the python side of numpy. Does this function need to be public?

Needs a docstring.